src/main/scala-sbt-0.13/com/typesafe/sbteclipse/core/Eclipse.scala

```diff
@@ -305,11 +305,22 @@ private object Eclipse extends EclipseSDTConfig {
       srcEntries <- srcEntriesIoSeq.toList.sequence;
       linkEntries <- srcLinkEntriesIoSeq.toList.sequence
     ) yield {
+      val eclipseOutputDirs: Seq[String] = for {
+        (fa: File, ta: Option[File]) <- srcDirectories
```

Comment:
can you rename `fa`, `ta`, and `dd` as well? I'm not sure what they refer to

Reply:
Ben,
After reviewing the code for awhile, I realized that I could simplify it by deriving the output directory from srcEntries rather than srcDirectories.   I was also better (hopefully) at naming variables.   Let me know what you think.
Phil